src/StorageSync/StorageSync/Common/StorageSyncClientCmdletBase.cs

```diff
@@ -13,13 +13,15 @@
 // ----------------------------------------------------------------------------------
 
 using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.ResourceManager.Common.Properties;
 using Microsoft.Azure.Commands.StorageSync.Common.Converters;
 using Microsoft.Azure.Commands.StorageSync.Common.Exceptions;
 using Microsoft.Azure.Commands.StorageSync.Interfaces;
 using Microsoft.Azure.Graph.RBAC.Version1_6_20190326.ActiveDirectory;
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Management.Automation;
```

Comment:
Is this necessary?

Reply:
yes

---
In reply to: [354639169](https://github.com/Azure/azure-powershell/pull/10652#discussion_r354639169) [](ancestors = 354639169)